From 875d17e3ffa036f2b942d9552379fd92ab6eec69 Mon Sep 17 00:00:00 2001
From: Bian Jin chen <kenjc.bian@rock-chips.com>
Date: Mon, 16 Sep 2019 11:42:26 +0800
Subject: [PATCH] arm64: configs: rockchip_defconfig: enable
 CONFIG_ION_SYSTEM_HEAP

Android 10 requires ion to allocate buffers for audio playback,
enable system heap by default to support it.

Change-Id: Id13e42254c7cfc2d07a33840d0dc4429e6fe7065
Signed-off-by: Bian Jin chen <kenjc.bian@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index a67ec6843f6c..8bac72bddeef 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -731,6 +731,7 @@ CONFIG_SYNC=y
 CONFIG_SW_SYNC=y
 CONFIG_SW_SYNC_USER=y
 CONFIG_ION=y
+CONFIG_ION_SYSTEM_HEAP=y
 CONFIG_ION_ROCKCHIP=y
 CONFIG_FIQ_DEBUGGER=y
 CONFIG_FIQ_DEBUGGER_NO_SLEEP=y
-- 
2.35.3

